packages/xo-server/src/api/xostor.mjs

```diff
@@ -152,9 +188,17 @@ export async function create({ description, disksByHost, force, ignoreFileSystem
     shared: true,
     type: 'linstor',
   })
+  const srUuid = xapi.getObject(srRef).uuid
```

Comment:
`this.getField` instead?

Reply:
That would best indeed because the new SR might not already be in object cache at this time.